feat(library): Introduce PHP as library (#24)
unikraft/catalog: library/php/8.2/README.md

> # Node 18 Image

Reviewer: ```suggestion
# PHP 8.2 Image
```

> This directory contains the definition for the `unikraft.org/node:18` image starting a simple HTTP server.

Reviewer: ```suggestion
This directory contains the definition for the `unikraft.org/php:8.2` image starting a simple HTTP server.
```

To run this image, [install Unikraft's companion command-line toolchain `kraft`](https://unikraft.org/docs/cli) and then you can run:

```console

> kraft run -p 8080:8080 unikraft.org/node:18

Reviewer: ```suggestion
kraft run -p 8080:8080 unikraft.org/php:8.2
```
